src/api/libcellml/module/libcellml

```diff
@@ -29,6 +29,7 @@ limitations under the License.
 #include "libcellml/analyserequationast.h"
 #include "libcellml/analysermodel.h"
 #include "libcellml/analyservariable.h"
+#include "libcellml/annotator.h"
 #include "libcellml/component.h"
```

Comment:
Does this need `enums.h` too?

Reply:
`enums.h` is included by `types.h` which in turn is included by most headers in this module. So, no need to include `enums.h`.

Reply:
If we are to add `enums.h` then why don't we add `types.h`?